container/flatten.bzl

```diff
@@ -14,7 +14,11 @@
 """A rule to flatten container images."""
 
 load(
-    "//container:layers.bzl",
+    "//skylib:path.bzl",
+    _get_runfile_path = "runfile",
+)
```

Comment:
Is this used?  trying to understand the diff.

Reply:
Sorry I was not familiar with the PR previously and I rebased my change instead of merging master. So this was unrelated change somehow got in.

Fixed right now.